src/components/templates/Audience/Audience.tsx

```diff
@@ -151,7 +142,9 @@ export const Audience: React.FC<AudienceProps> = ({ venue }) => {
     venue?.auditoriumColumns ?? DEFAULT_AUDIENCE_COLUMNS_NUMBER;
   const baseRows = venue?.auditoriumRows ?? DEFAULT_AUDIENCE_ROWS_NUMBER;
 
-  const [isAudioEffectDisabled, setIsAudioEffectDisabled] = useState(false);
+  const { isShown: isUserAudioOn, toggle: toggleUserAudio } = useShowHide(true);
```

Comment:
Boolean trap. To escape it, `useShowHide` should have options style parameter:
```
const { isShown: isUserAudioOn, toggle: toggleUserAudio } = useShowHide( { isShownByDefault: true });

Reply:
OOS
Created the issue + related changes sparkletown/internal-sparkle-issues#912